### PR TITLE
Bugfix for quantile regression

### PR DESCRIPTION
**Pull request by corwinjoy**
Thanks for writing this great wrapper!
I'd like to propose a minor bugfix. It turns out that quantile regression throws an exception when it tries to predict on one row. I have added a reshape call to fix this edge case.

### CODE REVIEW

File: skranger/ensemble/regressor.py [hunk 1]
```diff
@@ -310,6 +310,8 @@ def predict_quantiles(self, X, quantiles):
 
         forest = self._get_terminal_node_forest(X)
         terminal_nodes = np.array(forest["predictions"]).astype(int)
+        if len(terminal_nodes.shape) < 2:
+            terminal_nodes = np.reshape(terminal_nodes, (1, -1))
```
Comment (crflynn) [inline skranger/ensemble/regressor.py:314]:
```suggestion
        terminal_nodes = np.atleast_2d(terminal_nodes)
```
I think we can accomplish the same thing here with this, which is a bit simpler.

**Review by crflynn — CHANGES_REQUESTED** [review]
Thanks for catching this. I just requested one small change. Otherwise looks good.

File: skranger/ensemble/regressor.py (hunk 1 above)
Comment (corwinjoy) [reply skranger/ensemble/regressor.py]: Looks even better! Thanks for reviewing this.

Comment (crflynn) [reply skranger/ensemble/regressor.py]: Sure thing. Can you get rid of the if statement on the line before? After that I think we can merge.

Comment (corwinjoy) [reply skranger/ensemble/regressor.py]: Got it. I removed the unneeded if statement.

**Comment by crflynn** [comment]
Failure here is unrelated to your changes. I'll fix it and get it merged later today.

**Comment by corwinjoy** [comment]
Awesome! I did also add a unit test for this edge case so hopefully that will keep this issue from coming back as well.

**Review by crflynn — APPROVED** [review]
